Source/carthage/Build.swift

```diff
@@ -31,8 +31,8 @@ public struct BuildCommand: CommandType {
 
 				let (stdoutSignal, schemeSignals) = self.buildProjectInDirectoryURL(directoryURL, options: options)
 				stdoutSignal.observe { data in
-					stdoutHandle.writeData(data)
-					stdoutHandle.synchronizeFile()
+					stdoutHandle.write(data)
+					stdoutHandle.flush()
```

Comment:
I’m not sure we should be ignoring errors, but we were doing so before, too, so I guess it’s orthogonal to this PR.